openlibrary/plugins/upstream/account.py

```diff
@@ -253,9 +253,10 @@ def get_form(self) -> forms.RegisterForm:
 
     def get_recap(self):
         if self.is_plugin_enabled('recaptcha'):
-            public_key = config.plugin_recaptcha.public_key
-            private_key = config.plugin_recaptcha.private_key
-            return recaptcha.Recaptcha(public_key, private_key)
+        public_key = config.plugin_invisible_recaptcha.public_key  # Updated to use the new config entry
+        private_key = config.plugin_invisible_recaptcha.private_key  # Updated to use the new config entry
+        return recaptcha.Recaptcha(public_key, private_key)
+
```

Comment:
```suggestion
            public_key = config.plugin_invisible_recaptcha.public_key
            private_key = config.plugin_invisible_recaptcha.private_key
            return recaptcha.Recaptcha(public_key, private_key)
```
Indentation levels are meaningful in Python.  The `if` block code needs to be indented one level more than the `if` statement.
The comments can also be removed.  We can look at the code history in `git`, so these aren't really useful.